src/commands/meal.py

```diff
@@ -0,0 +1,29 @@
+from src.util.const import (
+    BREAKFAST, BREAKFAST_TEMPLATE,
+    DINNER_TEMPLATE,
+    MENU_MSG
+)
+from src.util.util import parse_menu
+from src.database import connect
+from src.util.const import NO_MENU_MSG
+import psycopg2.extras
+from datetime import date
+
+
+def handle_menu(meal):
+    def inner(update, context):
```

Comment:
We should probably put an assertion here to ensure `meal` is either `breakfast` or `dinner`